### PR TITLE
Add ArcSV 

### PR DESCRIPTION
**Pull request by xuxif**
We introduce a novel computational pipeline specifically designed to overcome existing limitations in archaic genome sequencing data. Our method capitalizes on the supporting reads of SV found in modern human genomes to infer the existence of SVs in archaic humans.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by xuxif** [comment]
@bioconda-bot  please add label

**Comment by xuxif** [comment]
@BiocondaBot Please approve it

**Comment by xuxif** [comment]
Hi, I was wondering if someone could take a look at this PR when they get a chance. Thank you!
@BiocondaBot

File: recipes/arcsv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - samtools
+    - bcftools
+    - bedtools
+test:
+  commands:
+    - ArcSV detect 
+
+about:
+  home: https://github.com/xuxif/ArcSV
+  license: GPL-3.0
```
Comment (mencian) [inline recipes/arcsv/meta.yaml:26]:
```suggestion
  license: GPL-3.0
  license_file: LICENSE
```

**Review by mencian — CHANGES_REQUESTED** [review]

File: recipes/arcsv/meta.yaml (hunk 1 above)
Comment (mencian) [reply recipes/arcsv/meta.yaml:26]: Please add and fill in the license_file field with the license packaged from source.

Comment (xuxif) [reply recipes/arcsv/meta.yaml:26]: @mencian  

Hello,

I've updated the meta.yaml and added the license.txt file as per the suggestions. Could you please review the changes in my pull request? I appreciate your assistance.

Thank you (3Q).

**Review by mencian — APPROVED** [review]

**Pull request merged**